frontend/www/js/omegaup/arena/arena.js

```diff
@@ -258,8 +259,25 @@ export class Arena {
 
     // Currently opened notifications.
     self.notifications = new Notifications();
+    self.clarificationNotifications = null;
     OmegaUp.on('ready', function() {
       self.notifications.attach($('#notifications'));
+      if (document.getElementById('clarification-notifications') !== null) {
+        self.clarificationNotifications = new Vue({
+          el: '#clarification-notifications',
+          render: function(createElement) {
+            return createElement('omegaup-notifications-clarifications', {
+              props: {
+                data: this.data,
+              },
+            });
+          },
+          data: { data: null },
```

Comment:
puedes darle un mejor nombre y evitar el uso de `null` porque un arreglo vacío es suficiente?

```suggestion
          data: { clarifications: [] },
```

Reply:
Listo, usaré el que sugieres de `initialClarifications`